mscore/shortcut.cpp

```diff
@@ -3783,6 +3794,13 @@ QString Shortcut::keysToString() const
                   s += "; ";
             s += Shortcut::keySeqToString(_keys[i], QKeySequence::NativeText);
             }
+      if (s.isEmpty() && _standardKey != QKeySequence::UnknownKey) {
```

Comment:
I would remove the use of _action and have a look to QKeySequence::keyBindings http://doc.qt.io/qt-5/qkeysequence.html#keyBindings